### PR TITLE
Custom network size

### PR DESCRIPTION
**Pull request by paurosello**
## Checklist

- [x] Update changelog in CHANGELOG.md.

The annotation can be used on:
- AWSCluster (affects the subnet of the masters)
- AWSMachineDeployment (affects the nodepool)

The subnet mask can be as big as the customer wants, min size is /28 imposed by the additional CIDR limit.

What I am not too convinced is having the annotation in AWSCluster and not AWSControlPlane, but it's the object we are using to reconcile the masters subnets currently

Also modified .gitignore due to generated files via live-debugging

### CODE REVIEW

**Review by njuettner — CHANGES_REQUESTED** [review]
LGTM, can we have a test for this?

File: service/controller/resource/ipam/create.go [hunk 1]
```diff
@@ -4,6 +4,9 @@ import (
 	"context"
 	"fmt"
 	"net"
+	"strconv"
+
+	awsoperatorannotation "github.com/giantswarm/aws-operator/pkg/annotation"
```
Comment (xh3b4sd) [inline service/controller/resource/ipam/create.go:9]: Can you put the import to the other project imports below? Also why not just `annotation` without the alias?

**Review by xh3b4sd — APPROVED** [review]
We maybe want to have a unit test for this. Would be cool to see this feature covered somehow, though I am not sure how straight forward it is to implement. We can chat about it if you like. ❤️

File: service/controller/resource/ipam/create.go (hunk 1 above)
Comment (paurosello) [reply service/controller/resource/ipam/create.go]: yeah, I got confused as initially that imported a different annotation package but it's fine now

**Comment by paurosello** [comment]
testing done in "structure" branch, merging this one :)

**Pull request merged**